frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -46,6 +46,11 @@ action.GitMachete.BaseFastForwardMergeBranchToParentAction.task-title=Fast-forwa
 action.GitMachete.BaseFastForwardMergeBranchToParentAction.task-subtitle=Fast-forwarding...
```

Comment:
Wowow so many changes in icons

Is it to keep them up to date with what's in latest versions of IntelliJ?

Reply:
Actually! while you're at this, pls also include https://github.com/VirtusLab/git-machete-intellij-plugin/issues/737

Aaand extract the icon changes unrelated to Merge action to a separate PR :machete:

Reply:
yup, we started to look a bit old 👴 Xd